api/v1/submissions/PKPSubmissionHandler.php

```diff
@@ -1673,4 +1681,34 @@ protected function getWriteDisabledErrors(string $schemaName, array $params): ar
 
         return $errors;
     }
+
+    /**
+     * Get a submissions's review assignment by id
+     *
+     * @param SlimRequest $slimRequest Slim request object
+     * @param APIResponse $response object
+     * @param array $args arguments
+     *
+     * @return APIResponse
+     */
+    public function getReviewAssignment($slimRequest, $response, $args)
+    {
+        $submission = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_SUBMISSION);
```

Comment:
I don't think submission will be set as an authorised context object. It's the scope of the submissions list (dashboard view). Can you double check if we'll need another way to determine if the subeditor has access to it?